### ISSUE TITLE
what's the reason about the large loss? 

### ISSUE THREAD

**Issue by TongWeiDP** [xy-guo/MVSNet_pytorch#19]
I used your pre-trained model，but from the beginning, the loss was very large. I checked the code, but no problem was found. What is the reason for this?and my learning rate was set 0.0001,Eagerly looking forward to your reply.Thanks.

<img width="588" alt="微信图片_20201209172324" src="https://user-images.githubusercontent.com/25115986/101610485-46cd1f80-3a43-11eb-92b7-1e566589e01b.png">
<img width="661" alt="3" src="https://user-images.githubusercontent.com/25115986/101610653-8136bc80-3a43-11eb-824f-ca973eadf47f.png">

**Comment by Fourpk** [comment]
您好，请问这个问题现在解决了吗，是什么原因呢
